CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 [Full changelog](https://github.com/mozilla/glean/compare/v26.0.0...master)
 
 * General:
+  * The baseline ping is now sent when the application goes to foreground, in addition to background and dirty-startup.
```

Comment:
Will you file bugs for the other language bindings?

Reply:
I thought we would handle iOS here -- and Python doesn't have these pings.